docs/index.rst

```diff
@@ -5,66 +5,85 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+Tahoe-LAFS is a Free and Open decentralized storage system.
+It distributes your data across multiple servers.
+Even if some of the servers fail or are taken over by an attacker,
+the entire file store continues to function correctly,
+preserving your privacy and security.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :caption: Introduction
 
    about
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Using Tahoe-LAFS
+
    INSTALL
```

Comment:
I've removed this article and another one (OS-X packaging) in my PR. I've also made some changes to Index file (brought all installation guides at the top). I am not sure how we resolve conflicts which will arise.

Reply:
I have updated the PR.

Reply:
Awesome!